utilities/driver-checksum.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
```

Comment:
```suggestion

# Takes in the source type of the driver, either gcp or file, a sha256 checksum, and the location of the file
# and checks if the sha256 checksum of the file actually matches the provided checksum.
# E.g ./driver-checksum.sh file 3ea61b22b3123871b203cd352c60884e336d63777145bfdad20388aa74b71a06 ../kernel-modules/container/kernel-modules/collector-ebpf-4.18.0-477.13.1.el8_8.x86_64.o.g
```

Reply:
Perhaps add a comment explaining what the script is for and how to use it.

Reply:
I added a small comment on the purpose of the script and a `usage` function describing what each argument should be, hopefully that is enough 🙂

Reply:
Good idea.